### DISCUSSION TITLE
Notifications - Home Assistant

### DISCUSSION THREAD

**Discussion by timmo001** [timmo001/system-bridge#1746] [Ideas] [answered]
Notifications have been around for a while. We should utilise this with Home Assistant via the WebSocket

**Comment by timmo001** [comment, marked as answer]
https://github.com/timmo001/core/tree/systembridge-notify

https://github.com/home-assistant/core/pull/76802

https://github.com/home-assistant/home-assistant.io/pull/23774

**Comment by Iceman248** [comment]
What to put in for target? I've tried device name, IP, but don't know what to put in, can't find anything on it.

**Comment by Iceman248** [comment]
Following up on this. I am trying again to see what I can do with this app, but still cannot figure out how to use notifications to run anything.
